src/class/ServerManager/ServerManager.cpp

```diff
@@ -0,0 +1,157 @@
+
+#include <ServerManager/ServerManager.hpp>
+
+/* TEST HANDLERS */
+
+Poll::ret_stt client_write(Poll &p, int fd, int revents, Poll::Param &param)
+{
+	(void)p;
+	(void)fd;
+	(void)revents;
+	(void)param;
+	try
+	{
+		param.req.sendResponse();
+	}
+	catch (const std::exception &e)
+	{
+		std::cerr << e.what() << '\n';
+		return Poll::DONE;
+	}
+
+	if (param.req.getBytesSent() < param.req.getResponse().size() ||
+		param.req.getResponseAttempts() >= HTTPRequest::MAX_CHUNK_ATTEMPTS)
+		return Poll::CONTINUE;
+	param.req.terminate();
+	return Poll::DONE;
+}
+
+Poll::ret_stt client_read(Poll &p, int fd, int revents, Poll::Param &param)
+{
+	(void)p;
+	(void)fd;
+	(void)revents;
+	(void)param;
+	if (param.req.getRequestAttempts() >= HTTPRequest::MAX_CHUNK_ATTEMPTS)
```

Comment:
i don't think I understand the logic of CHUNK_ATTEMPTS.
is it about trying to read but receiving nothing?

Reply:
I deleted it from now, it was something I thought nginx did while I was doing tests